docs/fundamentals/windows.md

```diff
@@ -111,19 +111,29 @@ Multiple windows can be simultaneously opened on Android, iOS on iPad (iPadOS),
 
 ```csharp
 Window secondWindow = new Window(new MyPage());
-Application.Current.OpenWindow(secondWindow);
+Application.Current!.OpenWindow(secondWindow);
```

Comment:
would it be better to do `Application.Current?.OpenWindow(secondWindow);`

I think we should avoid ! bad practice IMHO

Reply:
Done!

Reply:
i mean probably we would want to check for null ahead of time. 

Using ? is only really for unit testing the code. If you are running the app... the app exists and it will not be null, so there is no failing or hiding anything.